### PR TITLE
Fix incorrect shape issue when loading raw CIFAR-10 data

### LINKED ISSUE #1961: Incorrect shape when loading raw CIFAR-10 data
**Describe the bug**
When loading the CIFAR-10 data with the raw flag set to true (`art.utils.load_cifar(raw=True`), the labels (`y_train` and `y_test`) are formatted in the shape `(num_samples, 1)` which is non-standard. Other datasets such as MNIST load this as shape `(num_samples,)` which is standard since it should be a single vector of labels. This causes issues since the second dimension will make other functions believe this is one-hot encoded.

This issue occurs due to the two problematic lines
```python
y_train = np.reshape(y_train, (len(y_train), 1))
y_test = np.reshape(y_test, (len(y_test), 1))
```
and can be fixed by removing them.

**To Reproduce**
```python
from art.utils import load_cifar10, load_mnist

(x_train, y_train), (x_test, y_test), min_, max_ = load_cifar10(raw=True)
print(y_train.shape)  # (50000, 1)
print(y_test.shape)  # (10000, 1)

(x_train, y_train), (x_test, y_test), min_, max_ = load_mnist(raw=True)
print(y_train.shape)  # (60000,)
print(y_test.shape)  # (10000,)
```

**Expected behavior**
The labels should be of shape `(num_samples,)` rather than `(num_samples, 1)`.

**System information (please complete the following information):**
 - OS
 - Python version
 - ART version or commit number
 - TensorFlow / Keras / PyTorch / MXNet version

### PR DESCRIPTION
**Pull request by f4str**
# Description

Remove the lines in `art.utils.load_cifar10` which reshape the CIFAR-10 data to a non-standard size.

Fixes #1961 

## Type of change

Please check all relevant options.

- [ ] Improvement (non-breaking)
- [x] Bug fix (non-breaking)
- [ ] New feature (non-breaking)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Testing

Testing is trivial, just load the data and check for the shape.

```python
from art.utils import load_cifar10, load_mnist

(x_train, y_train), (x_test, y_test), min_, max_ = load_cifar10(raw=True)
print(y_train.shape)  # (50000,) instead of (50000, 1)
print(y_test.shape)  # (10000,) instead of (10000, 1)
```

**Test Configuration**:
- OS
- Python version
- ART version or commit number
- TensorFlow / Keras / PyTorch / MXNet version

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by beat-buesser — APPROVED** [review]
Hi @f4str Thank you very much!
